### DISCUSSION TITLE
How to upgrade ?

### DISCUSSION THREAD

**Discussion by Random-Ultra** [javinizer/Javinizer#356] [Q&A]
So this is probably a really stupid question, but how do I update to the newest release ?
I tried using the update command shown "Javinizer -UpdateModule"  But I get "WARNING: You already have the latest version of Javinizer! [2.5.10]"

Any help would be great, Thanks

**Comment by dgoryeo** [comment]
For me the workaround was to run power shell in admin mode and the command: `Update-Module Javinizer -Force`
